### PR TITLE
Add basic nextflow_schema.json

### PR DESCRIPTION
**Pull request by robsyme**
We noticed that the nf-gwas workflow did not have a `nextflow_schema.json` file that describes the workflow parameters in a formalized way. 

Including this json file allows external tools to understand the workflow inputs and parameters. Given that the [nf-validation](https://github.com/nextflow-io/nf-validation) is nearing release, including this schema may make it easier to do automated parameter validation before the workflow starts.

### CODE REVIEW

**Comment by drpatelh** [comment]
Some minor tweaks coming up.

**Comment by drpatelh** [comment]
Created a PR to your branch @robsyme 
https://github.com/robsyme/nf-gwas/pull/1

We have written a blog to show how to create and stay on top of the schema during pipeline development. Once it has been created hopefully, it is quite low maintainence when adding/removing new parameters. As Rob mentioned, there is some other cool stuff on the horizon that we will be able to do with the schema in Nextflow pipelines.
https://seqera.io/blog/best-practices-for-deploying-pipelines-with-nextflow-tower/

TL;DR: If you install [nf-core/tools](https://nf-co.re/tools/#installation) you can use the `nf-core schema build` command in the root of the pipeline repo and that can load the schema up in a website for you to edit.

What the schema looks like when loaded in the Tower UI:
image

Be awesome if you can sanity check the changes we have made please @seppinho @lukfor 🙏🏽 We mostly scraped your existing documentation and pipeline code to create the schema. Think it's mostly there but some extra eyes wouldn't go amiss.

**Comment by drpatelh** [comment]
Hola! Be great if we can get this into an official release please 🙏🏽 We are running the pipeline via a fork on Tower at the moment which isn't ideal.

**Comment by drpatelh** [comment]
Thank you 🚀

**Comment by lukfor** [comment]
Thanks for your help! 🙌
